lib/sprockets/sass/v3/importer.rb

```diff
@@ -124,7 +124,7 @@ def call_processor_input(processor, context, input, processors)
         def handle_complex_process_result(context, result, processors)
           data = result[:data] if result.key?(:data)
           context.metadata.merge!(result)
-          context.metadata.delete(:data)
+          # context.metadata.delete(:data)
```

Comment:
Yes, this was a bug. when i made the importer i was looking at previous versions of sprockets, to be able to implement the "old" way of processing processors in a "new" way, and that ;ine should be removed in order for the Importer to work correctly. Otherwise it causes issues with other engines because the data that is returned by those engines is not being merged in the context, which caused the Importer to show the original data from the processed file

This fixes this bug. Is a huge bug but the fix was minor. This is why i felt like this pull request needs to be reviewed ASAP.